readyapi4j-core/src/main/java/com/smartbear/readyapi4j/execution/RecipeExecutor.java

```diff
@@ -12,13 +11,54 @@
 
 public interface RecipeExecutor {
 
+    /**
+     * Submit a recipe for asynchronous execution.
+     *
+     * @param jsonText recipe to be executed.
+     * @return an instance of <code>Execution</code> containing the current state of execution
+     */
+    Execution submitRecipe(String jsonText);
+
+    /**
+     * Submit a recipe for synchronous execution.
+     *
+     * @param jsonText recipe to be executed.
+     * @return an instance of <code>Execution</code> containing the execution result
+     */
+    Execution executeRecipe(String jsonText);
```

Comment:
i'm a little hesitant to duplicating/overloading the submit/executeRecipe methods since they simply convert one type of argument to the other which could be done "externally" - could we instead have something like a 

TestRecipe TestCaseSupport.parseJsonRecipe( String json )

method that could be called to convert a json string to a recipe that can be passed to the existing submit/execute methods!? @spindelmanne what do you think?